packages/atomic/src/pages/examples/folding.html

```diff
@@ -76,6 +86,365 @@
+                                height: 30px;
+                                width: 30px;
+                                margin-right: 16px;
+                                border-radius: 4px;
+                              }
+
+                              .slack-msg-result-section-title {
+                                display: flex;
+                                background-color: #f6f7f9;
+                                border: 1px solid #f6f7f9;
+                                border-radius: 8px;
+                                flex-grow: 1;
+                                padding: 8px 8px 0px 8px;
+                              }
+
+                              .slack-title-icon {
+                                display: flex;
+                                width: 100%;
+                                margin-top: 8px;
+                                padding-left: 32px;
+                              }
+
+                              .slack-result-date {
+                                color: #616061;
+                                box-sizing: border-box;
+                                font-style: normal;
+                                font-weight: 400;
+                                font-size: 12px;
+                              }
+
+                              .slack-result-author-name {
+                                font-size: 16px;
+                                margin-right: 4px;
+                                font-weight: bold;
+                              }
+
+                              .slack-result-replies-excerpt {
+                                margin: 4px 0 16px 0;
+                              }
+                            </style>
+                            <div class="slack-title-icon">
+                              <atomic-result-section-title class="slack-msg-result-section-title">
+                                <span>
+                                  <atomic-result-section-visual image-size="small">
+                                    <atomic-result-image
+                                      field="slackmessageauthorprofilepicture"
+                                      aria-hidden="true"
+                                    ></atomic-result-image>
+                                  </atomic-result-section-visual>
+                                </span>
+                                <div>
+                                  <div>
+                                    <span class="slack-result-author-name">
+                                      <atomic-result-text field="slackmessageauthor"></atomic-result-text>
+                                    </span>
+                                    <span>
+                                      <atomic-result-date
+                                        class="slack-result-date"
+                                        field="date"
+                                        format="h:mm A - MM/DD/YYYY"
```

Comment:
I guess that's subjective, but I think the MM/DD/YYYY and DD/MM/YYYY formats are confusing, I would prefer the ISO format